dist/common/sysctl.d/99-scylla-aio.conf

```diff
@@ -1,2 +1,2 @@
 # Raise max AIO events
-fs.aio-max-nr = 5578536
```

Comment:
But, why, 5578536 was such a nice round number! :-)
But seriously, what does this number do - does it allocate memory proportional to this number? Or is just a **limit**? If it's just a limit, can't we set it to infinity or close enough to infinity?